cmd/generate/main.go

```diff
@@ -19,6 +19,8 @@ func main() {
 	var (
 		sourceCodePath = flag.String("src-path", "",
 			"The path of source code your want to generate support from, it can be either a file or a directory. (required)")
+		directiveMapName  = flag.String("directive-map", "", `Name of the generated map variable.(required)`)
```

Comment:
```suggestion
		directiveMapName  = flag.String("directive-map", "", `Name of the generated map variable. (required)`)
```

Would this be more "self documenting" if the flag was `-variable` or `-variable-name`? The fact that the type is a map seems inconsequential.

Reply:
At the beginning I also used `directive-map-name` but I'm not sure if it was too long lol. Changed to `directive-map-name` and `match-func-name` in latest commit

Reply:
I think you ignored the bit where "map" isn't really necessary information to communicate, but that's OK I guess.

I'm not sure if you want to do anything to enforce that the resulting variable is unexported or just suggest to the caller that it shouldn't be.

Reply:
> I'm not sure if you want to do anything to enforce that the resulting variable is unexported or just suggest to the caller that it shouldn't be.

I think it's just suggestion? User may really want the variable exported sometimes(e.g. they want to iterate though the map in other package)

Reply:
Updated: add comments for `directive-map-name` and `match-func-name`